src/Common/Core/Impl/Collections/EnumerableExtensions.cs

```diff
@@ -113,5 +113,11 @@ public static class EnumerableExtensions {
                 }
             }
         }
+
+        public static void ForEach<T>(this IEnumerable<T> enumerable, Action<T> action) {
```

Comment:
Meh. Eric was always opinionated. There is one on `List<T>` and we are using it. And there used to be one when Linq was an extension.